charts/pega/README.md

```diff
@@ -759,14 +759,18 @@ The Helm charts also support an automated install or upgrade with a Kubernetes J
 
 For installations of the Pega platform, you must specify the installer Docker image and an initial default password for the `administrator@pega.com` user.
 
+Along with this, you can specify the imagePullPolicy here, an element that defines when an image should be pulled. It is defaulted to `IfNotPresent`, meaning an image will be pulled if it is "not present". However, depending on how images are built, it may be worthwhile to change it to `Always`, as fixes may come out that aren't automatically applied to the image. 
```

Comment:
a few tweaks and still need more information about where the deployment pulls the images...please adjust>>
For installations of the Pega platform, you must specify the installer Docker image and an initial default password for the `administrator@pega.com` user.

Along with this, you can specify the imagePullPolicy here, an element that defines when an image should be pulled. It is defaulted to `IfNotPresent`, meaning an image will be pulled if it is "not present". However, depending on how images are built, it may be worthwhile to change it to `Always`, as fixes may come out that aren't automatically applied to the image.
change to>>
To install Pega Platform, you must specify the Pega installer Docker image that you downloaded and placed in your repository. Add an initial default password for the `administrator@pega.com` user.

You can specify a `imagePullPolicy`, which is a parameter that defines a use case for when a deployment pulls a Pega installer image from "where??????". By default, the policy is set to `IfNotPresent`; this specifies that the Pega installer image will be pulled if it is "not present" in your specified repository, defined using the `which-one????` parameter. Supported use cases are `IfNotPresent` and `Always`.
Change `imagePullPolicy` to `Always`, to configure your deployment to automatically pull any new Docker images that come out from "where???".

Reply:
@taz-pega-work `imagePullPolicy` is a standard kubernetes feature related to when to pull the docker image from the docker repository for a container included in a deployment or job. I don't think we want to document it ourselves.